dcos_launch/platforms/aws.py

```diff
@@ -662,21 +662,21 @@ def get_bootstrap_ip(self):
                                                  'us-west-2': 'ami-0116dcbe0583de7ca'},
     'cent-os-7.4-with-docker-selinux-enforcing': CENTOS_74_WITH_DOCKER_SELINUX_ENFORCING,
     # cent-os-7-dcos-prereqs runs centos 7.5. Taken from https://github.com/dcos/dcos-images/blob/master/centos/7.5/aws/DCOS-1.12.1/docker-1.13.1.git8633870/selinux_enabled_enforcing/dcos_images.yaml # noqa
```

Comment:
@jbarrick-mesosphere  please replace this comment from the actual location of these AMIs.

It is helpful for our reference.

Reply:
Fixed!